modules/tfm/tfm/boards/board/tfm_peripherals_config.h

```diff
@@ -173,14 +173,6 @@ extern "C" {
 #define TFM_PERIPHERAL_DPPI_CHANNEL_SECURE_MASK CONFIG_NRF_DPPI_CHANNEL_MASK_SECURE
 #endif
 
-#if defined(NRF5340_XXAA_APPLICATION)
-    #include <tfm_peripherals_config_nrf5340_application.h>
```

Comment:
Not relevant to this line but do we even use these headers anymore after this removal?

Reply:
Yes, in vanilla TF-M